docs/DeveloperGuide.adoc

```diff
@@ -412,7 +412,14 @@ Use case resumes at step 2.
 
 //
 [discrete]
-=== Use case: (UC07) Load and save data
+=== Use case: (UC07) Save updated Appointment data
+*MSS*
+
+1. User make changes in the appointment list, or a specific appointment via CRUD.
+2. System saves the updated data to the external text file
+3. System displays the updated appointment data in the main window
```

Comment:
Might be too specific.
How about writing
2. System saves the updated data
3. System displays the updated appointment data

Architecture diagram LGTM 👍

Reply:
I agree with Jun Hao on points 2 and 3 but otherwise it looks good :)

Reply:
Updated UC07 accordingly. Refer to 8d1aff4

Reply:
Merging pull request